### PR TITLE
add meta field, bug fixes and make sure foreign keys are added properly

### PR DESCRIPTION
**Pull request by realstraw**

### CODE REVIEW

**Review by schowsf — APPROVED** [review]
LGTM

**Review by damianxu88 — APPROVED** [review]
LGTM

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
- ThisBuild / version := "0.5.1"
```
Comment (damianxu88) [inline version.sbt:1]: @realstraw looks like branch is pointing to an older master, need to pull the latest

Comment (realstraw) [reply version.sbt]: thanks, fixed

**Review by damianxu88 — APPROVED** [review]
LGTM

**Pull request merged**